### PR TITLE
[jsk_robot_startup] move initialpose_publisher.l from jsk_pr2_startup/jsk_pr2_move_base

### PR DESCRIPTION
**Pull request by furushchev**

### CODE REVIEW

**Comment by k-okada** [comment]
please keep old filename, which just load new file and display deprecated and run the code

**Comment by furushchev** [comment]
@k-okada OK. Updated.

File: jsk_pr2_robot/jsk_pr2_startup/jsk_pr2_lifelog/initialpose_publisher.l [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+;; Author: Yuki Furuta <furushchev@jsk.imi.i.u-tokyo.ac.jp>
+
+(ros::roseus "initialpose_publisher")
+
+(ros::ros-warn "[DEPLICATED] initialpose_publisher.l is moved to jsk_robot_startup/util")
+
+(load "package://jsk_robot_startup/util/initialpose_publisher.l")
+
+(main)
+(exit)
```
Comment (k-okada) [inline jsk_pr2_robot/jsk_pr2_startup/jsk_pr2_lifelog/initialpose_publisher.l]: do we need `(main)` and `(exit)`?

Comment (furushchev) [inline jsk_pr2_robot/jsk_pr2_startup/jsk_pr2_lifelog/initialpose_publisher.l]: No. removed.

**Pull request merged**